index.html

```diff
@@ -272,7 +272,7 @@
 
 </section>
 
-# Introduction {#intro}
+# Privacy on the Web {#intro}
 
 <dfn>Privacy</dfn> is an essential value of the Web ([[?ETHICAL-WEB]],
```

Comment:
Privacy is not the only essential value of the web, and is in conflict with other values, such as 2.2 no harm, 2.5 security and 2.7 verifiable information. The intro needs to express a vision for how privacy can be pursued without compromising other co-equal essential values.

Reply:
Yes, we will be weaving that in.